### ISSUE TITLE
Update Firmware message, to large for Hardware Wallet Screen

### ISSUE THREAD

**Issue by stdevMac** [skycoin/hardware-wallet#176]
**Describe the bug**
When we run `go run cmd/cli/cli.go firmwareUpdate -f .../hardware-wallet/tiny-firmware/skycoin.bin`
The message resulting of a correct firmware updating is too large for the Hardware Wallet Screen

**Steps to Reproduce**

1. Go to the `skycoin\hardware-wallet` project
2. Run `make clean && make firmware`
3. Run `go run cmd/cli/cli.go firmwareUpdate -f path_to/skycoin/hardware-wallet/tiny-firmware/skycoin.bin` in `skycoin\harware-wallet-go` project.
4. Then follow the steps in the Hardware Wallet for a correct Updating process

**Actual behavior**
The actual message show when you run `firmwareUpdate` is 
```
"New firmware successfully installed."
"You may now unplug your Skycoin wallet."
```
From the wallet word just appear `wa`.

**Expected behavior**
The Hardware Wallet show a message according to the size of the screen.

**Possible implementation**
Change bootloader message for firmware update.
We should show a message like this:
```
"New firmware successfully installed."
"You may now unplug your SkyWallet."
```

**Comment by stdevMac** [comment]
According to this issue I think we should check all the messages showed in the `Hardware Wallet`.

**Comment by stdevAlDen** [comment]
@stdevMac duplicate of https://github.com/skycoin/hardware-wallet/issues/83?

**Comment by stdevMac** [comment]
@stdevAlDen Yes, this is duplicated..., I already solve the problem..

**Comment by stdevMac** [comment]
@stdevAlDen  Check the diff and give me some feedback

**Comment by olemis** [comment]
fixed in b894b63
